.github/workflows/build-nightly.yml

```diff
@@ -0,0 +1,153 @@
+name: Build (Nightly)
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+      - develop-pr-*
```

Comment:
We do not have such branches.

Reply:
This is on purpose - in case you wanna do a continuous test on a particular feature. In such a way you may cherry-pick commits into this branch for further testing.

Reply:
Thanks for your kind consideration. However, now the control flow of  `dae` takes PR and forked repo for this kind of situation (like dae-1).
Maybe we don't need this.